google/resource_compute_disk.go

```diff
@@ -495,3 +496,91 @@ func diskImageProjectNameEquals(project1, project2 string) bool {
 
 	return project1 == project2
 }
+
+func diskImageEquals(oldImageName, newImageName string) bool {
+	return strings.Contains(oldImageName, newImageName)
```

Comment:
What's the case where it's the image name that's a substring of the other one (as opposed to the family name that's a substring of the image name)?

Reply:
?

Reply:
Like, could this be `oldImageName == newImageName`?

Reply:
Ah I see :) Yes! Now that I added a specific one for for testing equality for family.

Done. I kept the `diskImageEquals` method even if it is trivial. I find that it reads better in `diskImageDiffSuppress`.

Reply:
Agreed, thanks :)